site/gatsby-site/src/pages/landingPage.js

```diff
@@ -37,7 +37,17 @@ const StyledCardParagraph = styled.p`
 `;
 
 const StyledRow = styled(Row)`
-  margin-bottom: 1em;
+  & + & {
```

Comment:
What is this `& + &` syntax doing?

Reply:
The `&` gives you access to the class name that styled-components will generate for the current component (kind of the `this` keyword). The `+` is just the adjacent sibling selector

Reply:
It does look cryptic tho